### PR TITLE
Must Gather Geneva Action

### PR DESCRIPTION
**Pull request by nilsanderselde**
Work in progress - any feedback would be appreciated. As suggested, I have started to reuse code from `oc`: https://github.com/openshift/oc/blob/release-4.3/pkg/cli/admin/mustgather/mustgather.go.

Not sure about the structure of this. It requires more code than the other current Geneva actions, so it's in its own package.

### CODE REVIEW

**Comment by jim-minter** [comment]
@nilsanderselde this seems to work.  Am willing to accept unit test as a follow-up - need to refactor the behaviour of the kubeactions struct as part of that.  Can you open an issue for that and pick it up?

File: pkg/frontend/frontend.go [hunk 1]
```diff
@@ -245,12 +251,9 @@ func (f *frontend) Run(ctx context.Context, stop <-chan struct{}, done chan<- st
...
 	f.s = &http.Server{
-		Handler:      middleware.Lowercase(f.setupRouter()),
-		ReadTimeout:  10 * time.Second,
-		WriteTimeout: time.Minute,
-		IdleTimeout:  2 * time.Minute,
-		ErrorLog:     log.New(f.baseLog.Writer(), "", 0),
-		BaseContext:  func(net.Listener) context.Context { return ctx },
+		Handler:     middleware.Lowercase(f.setupRouter()),
+		ErrorLog:    log.New(f.baseLog.Writer(), "", 0),
+		BaseContext: func(net.Listener) context.Context { return ctx },
```
Comment (asalkeld) [inline pkg/frontend/frontend.go:256]: what are the default timeouts - are they big enough? once this is removed are we relying on context timeouts?

File: pkg/util/version/version.go [hunk 2]
```diff
@@ -10,6 +10,7 @@ import (
 const (
 	OpenShiftVersion        = "4.3.9"
 	OpenShiftPullSpecFormat = "%s.azurecr.io/openshift-release-dev/ocp-release@sha256:f0fada3c8216dc17affdd3375ff845b838ef9f3d67787d3d42a88dcd0f328eea"
+	OpenShiftMustGather     = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:97ea12139f980154850164233b34c8eb4622823bd6dbb8e7772f873cb157f221"
```
Comment (asalkeld) [inline pkg/util/version/version.go:13]: this will need to be mirrored to acr

File: pkg/frontend/frontend.go (hunk 1 above)
Comment (jim-minter) [reply pkg/frontend/frontend.go:256]: by default there is no timeout.  This was more of a concern to me before we were firewalled; now that we are I think it's OK to drop these.

File: pkg/util/version/version.go (hunk 2 above)
Comment (jim-minter) [reply pkg/util/version/version.go:13]: It already is, and the crio layer handles the quay.io -> azurecr.io mapping for us.

**Pull request merged**